overlord/snapstate/autorefresh.go

```diff
@@ -496,6 +501,11 @@ func inhibitRefresh(st *state.State, snapst *SnapState, info *snap.Info, checker
 			// the error but don't change the snap state again.
 			return err
 		}
+		if _, ok := err.(*BusySnapError); ok {
+			st.Warnf(i18n.NG("refresh of snap %q was postponed for over %d day, refreshing anyway even though it is in use",
+				"refresh of snap %q was postponed for over %d days, refreshing anyway even though it is in use", days),
+				info.SnapName(), days)
+		}
 	}
```

Comment:
similar to above (I'm not totally happy with this, but it might prompt further discussion):
`snap %q activity has postponed its refresh for the maximum allowable %d days. It will now be refreshed.`

Reply:
maybe

```
snap %q has been running for the maximum allowable %d days since its refresh was postponed. It will now be refreshed.
```
?

Reply:
Applied (and above), thanks!